### PR TITLE
Minor change for getting the all and all_columns as booleans

### PR DESCRIPTION
**Pull request by imtj1**
@nancyxu123 
@jpechane

### CODE REVIEW

**Review by nancyxu123 — APPROVED** [review]

File: src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java [hunk 1]
```diff
@@ -75,11 +75,11 @@ public ChangeStreamSchema getStream(Timestamp timestamp, String streamName) {
 
             while (resultSet.next()) {
                 exist = true;
-                boolean allTables = isTrue(resultSet, 0);
```
Comment (nancyxu123) [inline src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java:78]: Small nit, was this function deleted?

**Comment by jpechane** [comment]
@imtj1 Could you please create a new Jira with the issue and align the commit messages to the new Jira id? The original Jira was already closed and released.
Thanksfor understanding

**Comment by imtj1** [comment]
> @imtj1 Could you please create a new Jira with the issue and align the commit messages to the new Jira id? The original Jira was already closed and released. Thanksfor understanding

Done

**Comment by jpechane** [comment]
@imtj1 Applied thanks!

**Pull request merged**